### DISCUSSION TITLE
Where can I download the old/new apps?

### DISCUSSION THREAD

**Discussion by igroti** [lichess-org/mobile#13] [Q&A]
Where can I download the old/new apps?

**Comment by veloce** [comment]
https://lichess.org/mobile

**Reply by igroti** [reply]
Thanks @veloce!

so that takes me to the current/“old” app

is there anywhere we can try the new one? (Aside from, say, downloading the whole codebase and compiling locally 😊)

I guess what I’m asking for is alpha/beta.

thanks!
